src/backend/joanie/tests/payment/test_backend_lyra.py

```diff
@@ -122,10 +127,14 @@ def test_payment_backend_lyra_create_payment_server_request_exception(self):
             body=RequestException("Connection error"),
         )
 
-        with self.assertLogs() as logger:
-            response = backend.create_payment(order, billing_address)
+        with self.assertRaises(PaymentProviderAPIException) as context:
+            with self.assertLogs() as logger:
+                backend.create_payment(order, billing_address)
```

Comment:
Those contexts can be merged:
```suggestion
        with self.assertRaises(PaymentProviderAPIException) as context, self.assertLogs() as logger:
            backend.create_payment(order, billing_address)
```

Reply:
thanks :ok_hand: